packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -132,43 +136,52 @@ export const prepareResolverKit = (
         registerTransaction(
           type: TxType,
           destinationAddress: AccountId,
-          amountValue: NatValue,
-        ): { vow: Vow<void>; txId: TxId } {
+          amountValue?: NatValue,
+        ): { result: Vow<void>; txId: TxId } {
           const txId: TxId = `tx${this.state.index}`;
           this.state.index += 1;
 
           const { transactionRegistry } = this.state;
           if (transactionRegistry.has(txId)) {
             trace(`Transaction already registered: ${txId}`);
-            return { vow: transactionRegistry.get(txId).vowKit.vow, txId };
+            return { result: transactionRegistry.get(txId).vowKit.vow, txId };
           }
```

Comment:
Now that the index nonce is managed in this method, the already registered case is no longer possible, right?

Do we have coverage for this case?